models/chef/Chef.model.js

```diff
@@ -0,0 +1,37 @@
+import { Schema } from 'mongoose';
+import { accountStatus } from '../../constants/chefEnums';
+
+const ChefSchema = new Schema(
+  {
+    userId: { type: ObjectId, ref: 'User', required: true },
```

Comment:
pay attention to this mongoose feature, it allow us to extend models: we can have common `User` model with `type` field with value `courier`, `chef`, `user` or `admin` and use this field later as a descriminator to extend `User` schema with additional fields for `Courier`, `Chef` and `Admin`
https://mongoosejs.com/docs/discriminators.html

discuss it with the team, it may be more convenient than populating Chef document with general User data on each query